shardingsphere-kernel/shardingsphere-authority/shardingsphere-authority-core/src/test/java/org/apache/shardingsphere/authority/provider/schema/DatabasePermittedPrivilegesProviderAlgorithmTest.java

```diff
@@ -41,6 +42,7 @@ public void assertBuildAuthorityRegistry() {
         Optional<ShardingSpherePrivileges> privileges = actual.findPrivileges(new Grantee("user1", "127.0.0.2"));
         assertTrue(privileges.isPresent());
         assertTrue(privileges.get().hasPrivileges("test"));
+        assertTrue(privileges.get().hasPrivileges(new DatabaseAccessSubject("*"), Collections.emptyList()));
```

Comment:
It's not a good test case, try:
```
assertTrue(privileges.get().hasPrivileges("db_dal_admin");
```
Because `db_dal_admin` is one of the two databases in your `user_database_mappings`.